src/plugins/presentation_util/common/labs.ts

```diff
@@ -21,30 +20,17 @@ export const solutionNames = ['canvas', 'dashboard', 'presentation'] as const;
  * provided to users of our solutions in Kibana.
  */
 export const projects: { [ID in ProjectID]: ProjectConfig & { id: ID } } = {
-  [TIME_TO_PRESENT]: {
-    id: TIME_TO_PRESENT,
-    isActive: false,
-    isDisplayed: false,
-    environments: ['kibana', 'browser', 'session'],
-    name: i18n.translate('presentationUtil.labs.enableTimeToPresentProjectName', {
-      defaultMessage: 'Canvas Presentation UI',
-    }),
-    description: i18n.translate('presentationUtil.labs.enableUnifiedToolbarProjectDescription', {
-      defaultMessage: 'Enable the new presentation-oriented UI for Canvas.',
-    }),
-    solutions: ['canvas'],
-  },
   [DEFER_BELOW_FOLD]: {
     id: DEFER_BELOW_FOLD,
     isActive: false,
     isDisplayed: true,
     environments: ['kibana', 'browser', 'session'],
     name: i18n.translate('presentationUtil.labs.enableDeferBelowFoldProjectName', {
-      defaultMessage: 'Defer loading below "the fold"',
+      defaultMessage: 'Defer loading panels below "the fold"',
     }),
     description: i18n.translate('presentationUtil.labs.enableDeferBelowFoldProjectDescription', {
       defaultMessage:
-        'Any Dashboard panels below the fold-- the area hidden beyond the bottom of the window, accessed by scrolling-- will not be loaded immediately, but only when they enter the viewport',
+        'Any panels below "the fold"-- the area hidden beyond the bottom of the window, accessed by scrolling-- will not be loaded immediately, but only when they enter the viewport',
```

Comment:
Do we need any clarity around the fact that this happens on dashboard?

Reply:
I figured that since the lab project is targeted to dashboard, and would *only* appear in the Dashboard Labs flyout, I assumed re-stating it was a Dashboard feature was redundant.

Reply:
Perfect. I forgot this was going in the dashboard labs specific flyout